### DISCUSSION TITLE
default 7F 3B 12

### DISCUSSION THREAD

**Discussion by DIOGO-DRH** [cedricp/ddt4all#856] [Q&A]
Bonjour, 
je rencontre un problème pour configurer les airbag ( désactiver suite a changement de sièges) sur une clio 2 phase 3. 
Apres avoir installer la dernière version 5.50 de ddt4all trouver sur le forum je me connecte avec un elm 327 usb : 
- Je sélectionne :  X65 - CLIO II
- je lance le scan des ecu 
- je sélectionne  AB81_X65
- configuration fonction 
- mode expert 
- je met toute les lignes en "sans" 
Puis au moment de configurer systeme j'ai ce code 
envoie requête ELM : 3B D6 00 20 00 00 (ROUGE)
Réponse ELM : 7F 3B 12

J'ai chercher dans le forum mais je n'est rien trouver. 

Merci 
Diogo

**Comment by Furtif** [comment]
> Bonjour, je rencontre un problème pour configurer les airbag ( désactiver suite a changement de sièges) sur une clio 2 phase 3. Apres avoir installer la dernière version 5.50 de ddt4all trouver sur le forum je me connecte avec un elm 327 usb :
> 
> * Je sélectionne :  X65 - CLIO II
> * je lance le scan des ecu
> * je sélectionne  AB81_X65
> * configuration fonction
> * mode expert
> * je met toute les lignes en "sans"
>   Puis au moment de configurer systeme j'ai ce code
>   envoie requête ELM : 3B D6 [00 20 00 00](tel:00 20 00 00) (ROUGE)
>   Réponse ELM : 7F 3B 12
> 
> J'ai chercher dans le forum mais je n'est rien trouver.
> 
> Merci Diogo

Sur le forum ? Vous avez la plus récente dans releases ici dans github

**Reply by Furtif** [reply]
https://github.com/cedricp/ddt4all/releases/tag/v2.1.1

**Reply by DIOGO-DRH** [reply]
il faut télécharger ddtall-win-installer-23.06.13-python-3.11.4-64bits.exe puis l'installer 
Ensuite installer source code zip ou tar.gs et le glisser dans le dossier ddtall ?

**Reply by Furtif** [reply]
Prenais la version qui vous convient si votre ordinateur c’est 64bit utilisez plutôt la version 64bits ou si 32bit l’autre puis copiez le fichier écu.zip dans c:\program files\ddt4all

**Reply by Furtif** [reply]
Il suffit d’installer et choisir créer l’icône sur bureau et voilà pour lancer utiliser l’icône du bureau

**Reply by Furtif** [reply]
Voir plus détails ici https://github.com/cedricp/ddt4all/discussions/845

**Comment by Furtif** [comment]
Je ne sais pas du sorts cette version la 5.50 🤔🤔🤨

**Comment by KarelSvo** [comment]
> * Je sélectionne :  X65 - CLIO II
> * je lance le scan des ecu
> *  je sélectionne AB81_X65

What is the scan result Diagversion,Supplier ,Soft,Version? In the CLIP database, the AB82 airbag is selected for Clio X65 phases 2, 3 and 4. UCE AIRBAG BOSCH AB8.2E
AB81 is for phase 1.

>   Réponse ELM : 7F 3B **12**

The sent command is not supported by your ecu   https://gitlab.com/py_ren/pyren/-/blob/master/pyren/mod_elm.py#L79



> Je ne sais pas du sorts cette version la 5.50 


This old complete installer and its previous versions have made ddt4all very popular due to its easy installation.  https://www.drive2.ru/b/498093336985338243
